gr-utils/python/modtool/tools/util_functions.py

```diff
@@ -57,15 +57,32 @@ def remove_pattern_from_file(filename, pattern):
 def str_to_fancyc_comment(text):
     """ Return a string as a C formatted comment. """
     l_lines = text.splitlines()
-    outstr = "/* " + l_lines[0] + "\n"
+    if len(l_lines[0]) == 0:
```

Comment:
Hm, while the above is actually a whitespace fix (in the sense: removes superfluous/erroneous whitespace from source code), this fixes code generation. OK if I split this out?